tests/OpenRealEstate.Transmorgrifiers.RealEstateComAu.Tests/ReaXmlTransmorgrifierTests.cs

```diff
@@ -118,6 +119,35 @@ public void GivenTheFileREAInvalidCharacterAndBadCharacterCleaning_Parse_ThrowsA
             result.UnhandledData.Count.ShouldBe(0);
         }
 
+        [Fact]
+        public void GivenTheFileREAHTmlInTitleAndDesc_Parse_ReturnsAListOfListingsWithHTMLRemovedFromTitleAndDescription()
+        {
+            // Arrange.
+            var reaXml = File.ReadAllText("Sample Data/REA-HTMLInTitleAndDesc.xml");

```

Comment:
We need a few more test cases, IMO.

So, just read in the normal sample file I use for most things.

Then, change the test to:

```
[Theory]
[ID("<b>Best listing ever!</b>", "<b>This is a great listing!</b>", "Best listing ever!", "This is a great listing!")] // Simple bold tags, removed.
[ID("The price of apples are < the price of <b>oranges</b>", "Hi Hi", "The price of apples a < the price of oranges", "Hi Hi")] // Random < symbol in title remains. No html in description.
...
```

Reply:
Yep, agree we need more test cases.

I thought about above, e.g reading one file and then 'changing' data based on the theory param, but.. the issue it's just text, so I can't "change" fields before parsing?

Ideas top of my head:
1. Have multiple sample files (one for each 'theory')
2. Have the one sample file, but with a placeholder, e.g "TITLE", and then do a string replace in the test

Thinking option 1 is the cleanest. Thoughts?

Reply:
Option 2.

[I've done that before, in this repo for some other tests](https://github.com/OpenRealEstate/OpenRealEstate.NET.Transmorgrifiers.RealestateComAu/blob/master/tests/OpenRealEstate.Transmorgrifiers.RealEstateComAu.Tests/ParseResidentialTests.cs#L287-L314). Less files, easier to 'grok' the actual test because we can see the values in the theory setup.

My general pattern here is:
- If I care about the element names, then a file.
- If I care about testing the _data_ of an element, then a single file.

(Disclaimer: the earlier tests might not follow this pattern, as I matured the architecture of this repo, over time. Like a good 🍷 . So I hope there more examples of this, than not.).

Reply:
No probs! Will follow whatever style you prefer in this repo.

Update coming soon.

Reply:
🔧 

I also had to update the regex, as it was failing for your second test (so good pickup!)